### DISCUSSION TITLE
请问大佬！执行morphTo2D从3d切换到2d，vc-heatmap热力图在2d下显示不了？切换回3d能正常显示？这个怎么解决？

### DISCUSSION THREAD

**Discussion by chenjiaobin** [zouyaoji/vue-cesium#529] [Q&A] [answered]
执行morphTo2D从3d切换到2d，vc-heatmap热力图在2d下显示不了？切换回3d能正常显示？这个怎么解决？

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/overlays/vc-overlay-heatmap
上面的在线文档能重新么？我看切换没问题呢？

**Reply by chenjiaobin** [reply]
我用的vue2版本https://zouyaoji.top/vue-cesium-v2/#/zh/extend/vc-heatmap（vue-cesium@2.4.1）
3d显示热力图如下
![image](https://github.com/zouyaoji/vue-cesium/assets/24197503/8529bb4c-495b-4a22-8761-5babc15b6b52)
2d热力图显示不了
![image](https://github.com/zouyaoji/vue-cesium/assets/24197503/ec6bf47e-61a2-4542-b485-388eb80accdb)

3d切2d是通过morphTo2D切换的

**Reply by zouyaoji** [reply]
额，vue2的没维护了啊，已经发布不了版本了。

**Reply by chenjiaobin** [reply]
请问，这个是vue2版本确实存在的问题是吧？大概知道啥问题吗，或者我可以怎么修改这个问题？能否告知下？

**Reply by zouyaoji** [reply, marked as answer]
不太清楚了，之前没人反馈过，你可以拉源码，用源码的文档加你的测试数据，然后调试看看，有问题的话，直接改源码 然后打包结果替换你项目上的node_modules下的vue-cesium。如果多人，可以打包结果放项目中，形成一个本地包，用pnpm从本地项目安装vue-cesium。

**Reply by chenjiaobin** [reply]
好的，谢谢，那我尝试下你的方式

**Reply by chenjiaobin** [reply]
排查了下，原来是我的viewer开启了scene3DOnly，关掉就好了

**Reply by zouyaoji** [reply]
好 找到原因就好
